promscale/installation/kubernetes/tobs.md

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
We don't need this here. As in the Tobs PR, the same file is added under Tobs directory.